examples/plot_model_card.py

```diff
@@ -72,19 +78,58 @@
 )
 license = "mit"
 
+eval_results = card.evaluate(
+    model, X_test, y_test, "r2", "random_type", "dummy_dataset", "tabular-regression"
+)
+
 card_data = CardData(
     license=license,
     tags=["tabular-classification"],
     datasets="breast-cancer",
+    eval_results=eval_results,
+    model_name="my-cool-model",
     metrics=["acc"],
 )
 
+# %% Adding metrics
+# ====================
+# We'll pass permutation importances, confusion matrix and classification report
+# to our model card template. Skops includes a util to calculate and parse
+# permutation importances, we'll use that. For confusion matrix and
+# classification report, we'll use tools from scikit-learn. Additionally, model
+# card template has an extra section for images, so we will use
+# ConfusionMatrixDisplay and put the created plot in that section.
+
+
+predictions = model.predict(X_test)
+permutation_importances = card.permutation_importances(model, X_test, y_test)
+confusion_matrix_arr = confusion_matrix(y_test, predictions, labels=model.classes_)
+clf_report = classification_report(y_test, predictions, labels=model.classes_)
+disp = ConfusionMatrixDisplay(
+    confusion_matrix=confusion_matrix_arr, display_labels=model.classes_
+)
+plt.savefig("./confusion_matrix.png")
```

Comment:
shouldn't save here, it should save in the model repo, maybe under `figs` folder?